facade/src/main/java/org/jboss/pnc/facade/providers/SCMRepositoryProviderImpl.java

```diff
@@ -97,8 +93,8 @@
     @Inject
     private Notifier notifier;
 
-    @Inject
-    private BpmManager bpmManager;
+    @Deprecated // use stateless approach
+    private AtomicInteger nextTaskId = new AtomicInteger(1);
```

Comment:
I don't think we need it. TaskIds were the way how to identify tasks in memory and Maitai AFAIK.